alibi_detect/tests/test_dep_management.py

```diff
@@ -95,23 +91,8 @@ def test_tensorflow_model_dependencies(opt_dep):
     dependency_map = defaultdict(lambda: ['default'])
     for dependency, relations in []:
         dependency_map[dependency] = relations
-    if opt_dep != 'all':
-        with pytest.raises(ImportError):
-            from alibi_detect.models import tensorflow as tf_models
-            check_correct_dependencies(tf_models, dependency_map, opt_dep)
-
-
-def test_torch_model_dependencies(opt_dep):
```

Comment:
Why do we no longer do the `models.pytorch` submodule tests?

Reply:
We will, see [this](https://github.com/SeldonIO/alibi-detect/pull/541#discussion_r913659850)